src/lib/lru/index.ts

```diff
@@ -0,0 +1,52 @@
+export class LRUCache<K, V> {
```

Comment:
Maybe use existing lru implementation?
eg https://www.npmjs.com/package/lru-cache

Reply:
replaced it with a module